Fix language setting description in `settingtypes.txt` (#14048)
minetest/minetest: builtin/settingtypes.txt

[*User Interfaces]

> #    Set the language. Set to (Use system language) to use system language.

Reviewer: > Since the text will be copied into `minetest.conf.example` when the strings are updated, is the phrase, "Set to (Use system language) ...," which is used in the setting dialog, fine for `minetest.conf`?

The wording proposed by this PR indeed doesn't make sense for `minetest.conf.example`.

What about this?

```suggestion
#    Set the language. By default, the system language is used.
```
